src/build.jl

```diff
@@ -14,6 +14,10 @@ function swap_our_param_with_jump_param(expr)
     end
 end
 
+"""This function takes an expression tree and replaces all instances of
+`JuMP.NLParameter` with the corresponding `ParameterRef`.
+"""
```

Comment:
I think it is the other way around :) Our parameters are removed, `JuMP.NLParameter` is swapped in, right?

Reply:
I commented on the previous commit, but oops, not what I had, or that either. But it's fixed now (ParameterRef to its value).